### PR TITLE
Audit comments

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
This PR standardizes some comments and fixes up some typos

### CODE REVIEW

File: src/commands/build/wranglerjs/output.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ use serde::Deserialize;
 use std::io::prelude::*;
 
 // This structure represents the communication between {wranglerjs} and
-// {wrangler}. It is send back after {wranglerjs} completion.
+// {wrangler}. It is sent back after {wranglerjs} completion.
 // FIXME(sven): make this private
```
Comment (ashleymichal) [inline src/commands/build/wranglerjs/output.rs]: i guess while we're at it
```suggestion
// TODO: (sven) make this private
```

File: src/commands/publish/upload_form/mod.rs [hunk 2]
```diff
@@ -57,7 +57,7 @@ pub fn build(
         }
         TargetType::Webpack => {
             log::info!("Webpack project detected. Publishing...");
-            // FIXME(sven): shouldn't new
+            // TODO: (sven) shouldn't new
```
Comment (EverlastingBugstopper) [inline src/commands/publish/upload_form/mod.rs]: @xtuc could you explain this comment?

File: src/commands/build/wranglerjs/mod.rs [hunk 3]
```diff
@@ -30,8 +30,8 @@ use std::thread;
 use std::time::Duration;
 
 // Run the underlying {wranglerjs} executable.
-//
-// In Rust we create a virtual file, pass the pass to {wranglerjs}, run the
+
+// In Rust we create a virtual file, pass it to {wranglerjs}, run the
 // executable and wait for completion. The file will receive the a serialized
```
Comment (EverlastingBugstopper) [inline src/commands/build/wranglerjs/mod.rs]:
```suggestion
// executable and wait for completion. The file will receive a serialized
```

File: src/commands/build/wranglerjs/mod.rs [hunk 4]
```diff
@@ -30,9 +30,9 @@ use std::thread;
 use std::time::Duration;
 
 // Run the underlying {wranglerjs} executable.
-//
-// In Rust we create a virtual file, pass the pass to {wranglerjs}, run the
-// executable and wait for completion. The file will receive the a serialized
+
+// In Rust we create a virtual file, pass it to {wranglerjs}, run the
+ // executable and wait for completion. The file will receive a serialized
```
Comment (ashleymichal) [inline src/commands/build/wranglerjs/mod.rs]:
```suggestion
// executable and wait for completion. The file will receive a serialized
```

File: src/commands/publish/upload_form/mod.rs (hunk 2 above)
Comment (ashleymichal) [reply src/commands/publish/upload_form/mod.rs]: i think this should actually be above ln 62, where we call `wranglerjs::Bundle::new` but i'm not certain precisely why.

File: src/commands/build/wranglerjs/mod.rs (hunk 4 above)
Comment (ashleymichal) [reply src/commands/build/wranglerjs/mod.rs]: this will fix the fmt error in CI

Comment (EverlastingBugstopper) [reply src/commands/build/wranglerjs/mod.rs]: thanks!!!

File: src/commands/publish/upload_form/mod.rs (hunk 2 above)
Comment (xtuc) [reply src/commands/publish/upload_form/mod.rs]: My idea was that each backends (rust, webpack, js) returns a Bundle/WorkerBundle/Worker that contains all the bindinds. The creation of the form (code here), would consume it and build a form regardless of the backend used.

If that was implemented building a new bundle wouln't make sense. However, wranglerjs's bundle doesn't contains any state (only constants) so it's fine.

Feel free to remove the comment tho. We could also inline the constants for now.

File: src/commands/publish/upload_form/mod.rs [hunk 5]
```diff
@@ -58,6 +58,7 @@ pub fn build(
         TargetType::Webpack => {
             log::info!("Webpack project detected. Publishing...");
             // TODO: (sven) shouldn't new
+            // see https://github.com/cloudflare/wrangler/pull/846#discussion_r342845421
```
Comment (xtuc) [inline src/commands/publish/upload_form/mod.rs]: I think a dedicated issue would be safer and easier to discuss in

**Pull request merged**